seedfarmer/commands/_parameter_commands.py

```diff
@@ -225,3 +228,14 @@ def _module_metatdata(
                 value=parameter_value,
             )
     return None
+
```

Comment:
Looks good to me. If we do expand this feature we may want to parse the entire manifest **first** and substitute any env vars before `_load_parameter_value` or any other method.